packages/grub-artifacts/build.yaml

```diff
@@ -0,0 +1,13 @@
+requires:
+  - name: "cos"
+    category: "system"
+    version: ">=0"
+
+
+unpack: true
+
+includes:
+  - ^/usr$
```

Comment:
It looks good :+1:  I'm just thinking that _maybe_ it makes more sense to have the files in `/usr/share/grub2` in the package as top level instead of `/usr/share/*`.  Because looking at https://github.com/rancher-sandbox/cOS-toolkit/blob/master/images/img-builder.sh#L33 seems the file are just consumed like "blobs"

Reply:
ahh that is a good point